aws/files/config/bootstrap.xml

```diff
@@ -443,6 +455,36 @@
                   </hip-profiles>
                   <action>allow</action>
                 </entry>
+                <entry name="Allow Scoreboard" uuid="48d6db71-1f6b-4a5c-996e-fa8ce2aa3abf">
+                  <to>
+                    <member>any</member>
+                  </to>
+                  <from>
+                    <member>untrust</member>
+                  </from>
+                  <source>
+                    <member>any</member>
+                  </source>
+                  <destination>
+                    <member>10.0.0.10</member>
+                  </destination>
+                  <source-user>
+                    <member>any</member>
+                  </source-user>
+                  <category>
+                    <member>any</member>
+                  </category>
+                  <application>
+                    <member>any</member>
```

Comment:
This is most likely either `web-browsing` or `ssl`.  You can see what application is being detected from the traffic logs.

Reply:
> This is most likely either `web-browsing` or `ssl`. You can see what application is being detected from the traffic logs.

Ah ok, I wound up just making one up. Any advantage to using existing?